bin/chpi_snr.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Plot signal-to-noise of continuous HPI coils as a function of time.
+Works by fitting a general linear model (HPI freqs, line freqs, DC, slope) to
+the data, and comparing estimated HPI powers with the residual (=variance
+unexplained by the model).
+Window length for SNR estimates can be specified on the command line.
+Longer windows will by nature include more low frequencies and thus have
+larger residual variance (lower SNR).
+
+Tested with Python 2.7, MNE 0.11.0
```

Comment:
Don't really need this line, it will change over time anyway

0.11 is pretty old now BTW, I recommend using 0.12 or even latest `master` if you're going to be developing